### DISCUSSION TITLE
월드 오류 팝업

### DISCUSSION THREAD

**Discussion by jun5640** [naverz/zepeto-studio-kor#193] [5. Q&A]
월드를 퍼블리싱 한 후에 일부 유저에게 위에 작은 팝업으로 빨간 글씨로 일시적인 오류가 발생했습니다. 라는 오류가 발생한다고 전달받았습니다.

혹시 해당 오류가 어떠한 이슈로 발생하는 오류인지 알 수 있을까요?

**Comment by beanyh** [comment]
어떤 팝업인지 화면이나 메시지 공유 가능하실까요? 증상 파악에 도움이 될 것 같아요~

**Reply by jun5640** [reply]
![photo_2022-01-11_13-30-31](https://user-images.githubusercontent.com/13131384/148881466-98e57d5c-ec6b-4310-95f1-850e1a2f0c87.jpg)

위 사진과 같은 오류메시지가 나왔습니다.

**Reply by beanyh** [reply]
확인해보겠습니다 com.posco.park1538

**Reply by beanyh** [reply]
앱을 완전히 종료 후 재접속 시 정상적으로 입장이 되고 있습니다. 
내부적으로 찾아봤는데 특별한 이유가 발견되지는 않아서, 우선적으로는 오류가 발생하는 분들에게 '앱종료 후 재입장'을 안내 해주셔야 할 듯 합니다. 
추가적으로 확인되는 사항 있으면 공유 드리겠습니다.
